src/server/internalServer.cpp

```diff
@@ -1033,6 +1035,11 @@ std::unique_ptr<Response> InternalServer::handle_content(const RequestContext& r
            + suggestSearchMsg(searchURL, kiwix::urlDecode(pattern));
   }
 
+  const std::string archiveUuid = getArchiveId(*archive);
```

Comment:
`getArchiveId` (and others) has been introduced by 8b12434 to avoid duplication between `kiwix::Reader::getId` and `kiwix::Book::update` which now use the `zim::Archive`.

Now that `kiwix::Reader` has been removed (and `getArchiveId` implementation simplified) you would tend to directly use `archive.getUuid()`

Reply:
Done